web_external/templates/body/submissionPage.jade

```diff
@@ -18,7 +18,7 @@
   if submission.has('overallScore')
     .c-submission-overall-score-container
       .c-pre-text Overall score:
-      .c-overall-score= Math.round(submission.get('overallScore') * 1000) / 1000
+      .c-overall-score= submission.get('overallScore').toFixed(3)
```

Comment:
Yes, intentional. I find this:

    0.683
    0.680
    0.665

easier to read than:

    0.683
    0.68
    0.665

However, by itself it looks like toFixed() doesn't always round as expected, so this should probably be the original line with toFixed() applied at the end.

Still, it looks like the `Math.round(x * y) / y` construct won't always round as expected due to floating point precision inaccuracies. http://stackoverflow.com/a/23204425/2522042 points to a 'Decimal Rounding' example on MDN that might be even better.